### ISSUE TITLE
Three Layer CNN With 90.33% accuracy

### ISSUE THREAD

**Issue by meghanabhange** [zalandoresearch/fashion-mnist#91]
I tried to implement a three-layered CNN with Batch Normalisation and MaxPooling. I got an accuracy of 90.33 % after 5000 iterations. I used Batch normalisation after every layer to accelerate the performance.

Also, I got an accuracy of 99.04 % on MNIST with the same network 

The Architecture is as follows- 
1.Convolutional Layer 1 with output feature map 16 and 5*5 kernel ReLU Activation

- MaxPooling 1

- BatchNormalisation 1

2. Convolutional Layer 2 with output feature map 32 and 5*5 kernel ReLU Activation

- MaxPooling 2

- BatchNormalisation 2

3. Convolutional Layer 3 with output feature map 64 and 5*5 kernel ReLU Activation

- MaxPooling 3

- BatchNormalisation 3

4. Fully Connected Layer.

[Code](https://github.com/meghanabhange/FashionMNIST-3-Layer-CNN)
